invenio_records_marc21/ui/theme/assets/semantic-ui/js/invenio_records_marc21/user_dashboard/index.js

```diff
@@ -80,6 +81,7 @@ const initSearchApp = createSearchAppInit({
   "ResultsList.item": Marc21RecordResultsListItem,
   "SearchApp.facets": ContribSearchAppFacetsWithConfig,
   "SearchApp.searchbarContainer": RDMRecordSearchBarContainer,
+  "SearchBar.element": RDMRecordSearchBarElement,
```

Comment:
what is this line for in a translation commit?

Reply:
This adds the RDMSearchBar element to the user dashboard and replaces the button labeled 'Search,' which is not translated, with an icon.